docs/competency-api/employability.md

```diff
@@ -4,4 +4,51 @@ title: Employability
 nav_order: 6
 parent: Competency API
 ---
-# Employability
\ No newline at end of file
+# Employability
+
+## Update Employability
+
+### Description
+
```

Comment:
Needs a description

Reply:
Yeah a short description of what the route does and when it can be used wouldnt hurt.